### PR TITLE
IGNORE : COVID-19 India Data

### PR DESCRIPTION
**Pull request by thejeshgn**
[Go to 316](https://github.com/datacommonsorg/data/pull/316)

### CODE REVIEW

File: scripts/covid19_india/cases_count_states_data/preprocess.py [hunk 1]
```diff
@@ -14,13 +14,14 @@
...
 import csv
+import pandas as pd
 import urllib.request
 
 # 1. All counts are at State/UT level
 # 2. Only cumulative counts are available at this point
 # 3. 'unassigned' cases are ignored
 # 5. Wrong state codes are ignored
-# 4. Sometimes reports are published more than once
+# 4. Sometimes reports are published more than once. The we take the last report on that day
```
Comment (pradh) [inline scripts/covid19_india/cases_count_states_data/preprocess.py:24]: nit: Then

File: scripts/eurostat/regional_statistics_by_nuts/education_enrollment/README.md [hunk 2]
```diff
@@ -70,8 +70,11 @@ The license is [available](https://ec.europa.eu/eurostat/about/policies/copyrigh
 #### Template MCFs
```
Comment (pradh) [inline scripts/eurostat/regional_statistics_by_nuts/education_enrollment/README.md:70]: I'm puzzled about the diffs to scripts/eurostat dir. Do you need to sync to latest?

File: scripts/covid19_india/cases_count_states_data/preprocess.py [hunk 3]
```diff
@@ -40,32 +41,50 @@
...
-                "CumulativeCount_MedicalConditionIncident_COVID_19_PatientRecovered"] = row[
-                    "value"]["cured"]
-            processed_dict[
-                "CumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased"] = row[
-                    "value"]["death"]
 
-            writer.writerow(processed_dict)
+    df = pd.json_normalize(data["rows"])
+
+    #convert to datetime and the drop the not required columns
```
Comment (pradh) [inline scripts/covid19_india/cases_count_states_data/preprocess.py:55]: nit: and then drop...

**Review by pradh — APPROVED** [review]
Thanks for the fixes!

File: scripts/eurostat/regional_statistics_by_nuts/education_enrollment/README.md (hunk 2 above)
Comment (thejeshgn) [reply scripts/eurostat/regional_statistics_by_nuts/education_enrollment/README.md:70]: New clean [PR](https://github.com/datacommonsorg/data/pull/316). Closing this one.

**Pull request closed without merging**